### PR TITLE
Ghosts no longer drift in space

### LINKED ISSUE #26359: Ghosts drift in space
Who was that added Moved recently.

### PR DESCRIPTION
**Pull request by coiax**
Fixes #26359.

:cl: coiax
fix: Ghosts no longer drift in space.
/:cl:

Readds Moved() to ghosts.

### CODE REVIEW

**Comment by MrStonedOne** [comment]
so, have you checked (and confirmed no side effects) every other Moved() in code to ensure no other side effects?

**Comment by coiax** [comment]
The only proc in the chain of /mob/dead/observer Moved is /atom/movable/Moved

**Comment by Cyberboss** [comment]
PROXY SENSORS!

File: code/modules/mob/dead/observer/observer.dm [hunk 1]
```diff
@@ -808,6 +808,10 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 	if(!invisibility)
 		to_chat(user, "It seems extremely obvious.")
 
+// Ghosts have no momentum, being massless ectoplasm
```
Comment (MrPerson) [inline code/modules/mob/dead/observer/observer.dm]: yo momma so fat, her ghost drifts through space

**Comment by coiax** [comment]
DNM because change is pointless while ghosts do not call Moved

**Comment by MrStonedOne** [comment]
coaix, 

```
git fetch upstream pull/26435/head:pr-26435
git checkout fix-26359
git rebase pr-26435
```

no need for two prs

**Comment by coiax** [comment]
It's no longer DNM, stop tagging things you rude bot.

**Comment by Cyberboss** [comment]
Bot won't remove DNM and WIP it only adds them

**Pull request merged**